extensions/amp-analytics/0.1/test/test-video-analytics.js

```diff
@@ -74,57 +83,42 @@ describes.realWin(
           'selector': selectors,
         };
 
-        data = {
-          'autoplay': false,
-          'currentTime': 0.0045,
-          'duration': 15,
-          'height': 399,
-          'id': 'myVideo',
-          'muted': false,
-          'playedTotal': 0.0045,
-          'playedRangesJson': '[[0,0.0045]]',
-          'state': 'playing_manual',
-          'width': 700,
-        };
-        data2 = {
-          'autoplay': false,
-          'currentTime': 0.0045,
-          'duration': 15,
-          'height': 399,
-          'id': 'myVideo-2',
-          'muted': false,
-          'playedTotal': 0.0045,
-          'playedRangesJson': '[[0,0.0045]]',
-          'state': 'playing_manual',
-          'width': 700,
-        };
-        dataPercentagePlayed = {
-          'autoplay': false,
-          'currentTime': 0.0045,
-          'duration': 15,
-          'height': 399,
-          'id': 'myVideo',
-          'muted': false,
-          'normalizedPercentage': '100',
-          'playedRangesJson': '[[0,0.0045]]',
-          'playedTotal': 0.0045,
-          'state': 'paused',
-          'width': 700,
-        };
-
-        dataPercentagePlayed2 = {
-          'autoplay': false,
-          'currentTime': 0.0045,
-          'duration': 15,
-          'height': 399,
-          'id': 'myVideo-2',
-          'muted': false,
-          'normalizedPercentage': '50',
-          'playedRangesJson': '[[0,0.0045]]',
-          'playedTotal': 0.0045,
-          'state': 'paused',
-          'width': 700,
-        };
+        function makeData(id, isPercentage, percent) {
+          if (isPercentage) {
+            return {
+              'autoplay': false,
+              'currentTime': 0.0045,
+              'duration': 15,
+              'height': 399,
+              'id': id,
+              'muted': false,
+              'normalizedPercentage': percent,
+              'playedRangesJson': '[[0,0.0045]]',
+              'playedTotal': 0.0045,
+              'state': 'paused',
+              'width': 700,
+            };
+          } else {
```

Comment:
nit: omit the `else` clause. Just return the object:
```
if (isPercentage) {
  return Obj1
}
return Obj2
```

This is for readability (more indents is harder to read)